src/components/CollaboratorModal/components/MainSubmodal.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+  ModalHeader,
+  ModalBody,
+  Grid,
+  GridItem,
+  Text,
+  Box,
+  Divider,
+  FormControl,
+  Input,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalProps,
+  useFormControlContext,
+  Skeleton,
+  Stack,
+} from "@chakra-ui/react"
+import {
+  IconButton,
+  ModalCloseButton,
+  FormErrorMessage,
+  FormLabel,
+  Button,
+} from "@opengovsg/design-system-react"
+import _ from "lodash"
+import { useEffect, useState } from "react"
+import { FormProvider, useForm } from "react-hook-form"
+import { BiTrash } from "react-icons/bi"
+import { useParams } from "react-router-dom"
+
+import { useLoginContext } from "contexts/LoginContext"
+
+import * as CollaboratorHooks from "hooks/collaboratorHooks"
+
+import { Collaborator } from "types/collaborators"
+import { MiddlewareError } from "types/error"
+import { DEFAULT_RETRY_MSG, useSuccessToast } from "utils"
+
+import { ACK_REQUIRED_ERROR_MESSAGE } from "../constants"
+
+import { AcknowledgementSubmodalContent } from "./AcknowledgementSubmodal"
+
+const LAST_LOGGED_IN_THRESHOLD_IN_DAYS = 60
+
+const numDaysAgo = (previousDateTime: string): number => {
+  const currDateTime = new Date(Date.now())
+  const prevDateTime = new Date(previousDateTime)
+  return Math.floor(
+    (currDateTime.getTime() - prevDateTime.getTime()) / (60 * 60 * 24 * 1000)
+  )
+}
+
+interface CollaboratorListProps {
+  onDelete: (user: Collaborator) => void
+}
+
+const CollaboratorListSection = ({ onDelete }: CollaboratorListProps) => {
+  const { email } = useLoginContext()
+  const { siteName } = useParams<{ siteName: string }>()
+  const {
+    data: collaborators,
+    isError,
+  } = CollaboratorHooks.useListCollaboratorsHook(siteName)
+  const { isDisabled } = useFormControlContext()
+
+  return (
+    <Box mt="40px">
+      {collaborators?.map((collaborator: Collaborator) => (
+        <>
+          <Grid
+            templateColumns="repeat(11, 1fr)"
+            h="56px"
+            key={`collaborator-${collaborator.id}`}
+          >
+            <GridItem colSpan={8}>
+              <Box display="flex" alignItems="center" h="100%">
+                <Text>{collaborator.email}</Text>
+                <Text pl="4px" color="gray.500">
+                  {email === collaborator.email ? "(You)" : null}
+                </Text>
+                {numDaysAgo(collaborator.lastLoggedIn) >=
+                  LAST_LOGGED_IN_THRESHOLD_IN_DAYS && (
+                  <Text pl="4px" color="danger.500">
+                    {`(Last logged in ${numDaysAgo(
+                      collaborator.lastLoggedIn
+                    )} days ago)`}
+                  </Text>
+                )}
+              </Box>
+            </GridItem>
+            <GridItem colSpan={2}>
+              <Box display="flex" alignItems="center" h="100%">
+                <Text textTransform="capitalize">
+                  {_.capitalize(collaborator.role)}
+                </Text>
```

Comment:
Hmm I think we can skip the capitalisation from lodash and only rely on the CSS transformation?

Also, the role does not appear (see above screenshot). I think the data is in `collaborator.SiteMember.role` instead (similar to your `CollaboratorDto`)

Reply:
we can't use `text-transform` sadly - see [here](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform#:~:text=The%20text%2Dtransform%20CSS%20property,help%20improve%20legibility%20for%20ruby.)

the spec states 
>`capitalize`
Is a keyword that converts the first letter of each word to uppercase. Other characters remain unchanged (they retain their original case as written in the element's text).

Reply:
for data - added the data transformation in the hook. didn't expect this because ts wasn't throwing an error... structural typing do be like that